packages/medusa/src/api/routes/store/orders/lookup-order.ts

```diff
@@ -118,6 +120,10 @@ export class StoreGetOrdersParams {
   @IsEmail()
   email: string
 
+  @IsString()
```

Comment:
Todo: extends FindParams, the expand is also valid. It needs to be added in the oas as well. I let you check the other handlers as well 👍